### PR TITLE
Upgrades Logstash to 5.6.16

### PR DESCRIPTION
**Pull request by devlinmr**
Upgrades binary insstall and removes `document_type` in readiness for ES 6.

### CODE REVIEW

File: conf.d/20_output_journald_elasticsearch.conf [hunk 1]
```diff
@@ -34,7 +34,6 @@ output {
     elasticsearch {
       id => "journald_es"
       index => "journald-%ELASTICSEARCH_INDEX_SUFFIX%%{+YYYY.MM.dd}"
-      document_type => "%{transport}"
```
Comment (marcinc) [inline conf.d/20_output_journald_elasticsearch.conf:37]: We should capture transport and store it in another field for reference, now that we drop `document_type`.

**Review by marcinc — CHANGES_REQUESTED** [review]

File: conf.d/20_output_journald_elasticsearch.conf (hunk 1 above)
Comment (devlinmr) [reply conf.d/20_output_journald_elasticsearch.conf:37]: Agreed that this is not required

**Review by marcinc — APPROVED** [review]
LGTM.

**Comment by marcinc** [comment]
LG

**Pull request merged**